### PR TITLE
Open API Specs

### PR DESCRIPTION
**Pull request by epociask**
## Changes proposed
* Page rendering for OpenAPI spec @ https://base-org.github.io/pessimism/ 
* Built `swagger.yml` to represent/define Pessimism API contract

### CODE REVIEW

File: docs/openapi/swagger.yml [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+    It's also worth noting that this spec is not a complete representation of the application's current functionality.**
+  license:
+    name: MIT License
+    url: https://github.com/base-org/pessimism/blob/master/LICENSE
+  version: 0.0.1
+externalDocs:
+  description: Find out more about Pessimism's architecture
+  url: 'https://github.com/base-org/pessimism/docs/ARCHITECTURE.md'
+servers:
+  - url: 'http://127.0.0.1:8080'
```
Comment (anupsv-cb) [inline docs/openapi/swagger.yml:16]: just a thought, maybe worth putting in a placeholder external than localhost ?

File: docs/API.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+## Pessimism API
+
+### Overview
+The Pessimism API is a RESTful HTTP API that allows users to interact with the Pessimism application. The API is built using the [go-chi](https://github.com/go-chi/chi) framework and is served using the native [http package](https://pkg.go.dev/net/http). The API is designed to be modular and extensible, allowing for the addition of new endpoints and functionality with relative ease.
+
+Currently, interactive endpoint documentation is hosted via [Swagger UI](https://swagger.io/tools/swagger-ui/) at [https://pessimism.app/api/docs](https://pessimism.app/api/docs).
```
Comment (anupsv-cb) [inline docs/API.md:6]: There's a way to host swagger API through github pages. Maybe worth doing that?

Comment (epociask) [reply docs/API.md]: https://base-org.github.io/pessimism/

File: docs/openapi/swagger.yml (hunk 1 above)
Comment (epociask) [reply docs/openapi/swagger.yml]: Fixed!

**Pull request merged**